generators/javascript/procedures.ts

```diff
@@ -58,7 +59,9 @@ export function procedures_defreturn(block: Block, generator: JavascriptGenerato
       loopTrap + branch + xfix2 + returnValue + '}';
   code = generator.scrub_(block, code);
   // Add % so as not to collide with helper functions in definitions list.
-  generator.definitions_['%' + funcName] = code;
+  // TODO(#7600): find better approach than casting to any to override
```

Comment:
I would appreciate if this was linked to https://github.com/google/blockly/issues/7326 as well.

Reply:
I've added a reference to #7326 to #7600.